reuelcj/christa-reuel-ux-aop
# Cja 28 add a database

## Main Goal
Refactor your app to get its data from a live database using asynchronous requests.

### When Accomplished
- [x] The user data in the app now comes from a live online database.
- [x] The app's client-side interface continues to represent user data as defined in the [`localStorage` API spec](https://github.com/Banno/ux-intern-onboarding/blob/master/resources/specs/localStorage-api-spec.md)

### How To Test
Start server ` yarn start.`
Create User and check to see if it's added to the User List and edit and delete it.

## Checklist

Under penalty of public shaming, I avow that I:


Browsers tested:

## Dependencies
